### DISCUSSION TITLE
Is QMEU enabled for Linux runners for docker buildx?

### DISCUSSION THREAD

**Discussion by ajhalili2006** [cirruslabs/cirrus-ci-docs#793] [Q&A] [answered]
I would like to do `docker buildx` for my Docker images to ensure that Raspberry Pi and i836 users can still run our images without switching to amd64 ftom ARM-based CPUs.

But before we do that, do you enable QMEU on kernel-side of the Linux/macOS runners?

**Comment by fkorotkov** [comment, marked as answer]
Hey Andrei, 

For [privileged containers](https://cirrus-ci.org/guide/linux/#kvm-enabled-privileged-containers) Cirrus spins up a GCP VM with [nested virtualization](https://cloud.google.com/compute/docs/instances/enable-nested-virtualization-vm-instances) enabled but regular [`docker_builder`](https://cirrus-ci.org/guide/docker-builder-vm/) doesn't have nested virtualization enabled. It's a matter of a flag being added on our side. 🤔 So the answer to your question is "no, it's not but we can easily add the support".

Right now the KVM flavored `docker_builder` VM only installs `quemu-kvm`:

https://github.com/cirruslabs/vm-images/blob/eabac957f8d5e9af8091f9234e5ace5b0336bcd6/googlecompute/ubuntu_images.json#L104

Will you need any other packages?

**Reply by fkorotkov** [reply]
Created https://github.com/cirruslabs/cirrus-ci-docs/issues/794 which should cover this use case in more generic way.

**Reply by ajhalili2006** [reply]
@Thabks, @fkorotkov. I selected your reply as the answer to this one, and I'll also subscribe to that linked issue.
